package.json

```diff
@@ -22,12 +22,12 @@
   },
   "homepage": "https://github.com/seegno/angular-oauth2",
   "dependencies": {
-    "angular": "^1.4.0",
-    "angular-cookies": "^1.4.0",
+    "angular": "1.5.9",
```

Comment:
Yeah, it's probably better to do so, particularly if we wish to revisit this and find a workaround, it will make it easier to contextualize the issue.